Silence unsupported LLVM version warning.

This unbreaks several tests with Hadrian.

diff -upr ghc-9.4.4.orig/compiler/GHC/CmmToLlvm.hs ghc-9.4.4/compiler/GHC/CmmToLlvm.hs
--- ghc-9.4.4.orig/compiler/GHC/CmmToLlvm.hs	2023-03-06 18:50:58.351618309 +0100
+++ ghc-9.4.4/compiler/GHC/CmmToLlvm.hs	2023-03-06 18:53:49.483822141 +0100
@@ -62,12 +62,6 @@ llvmCodeGen logger cfg h cmm_stream
          debugTraceMsg logger 2
               (text "Using LLVM version:" <+> text (llvmVersionStr ver))
          let doWarn = llvmCgDoWarn cfg
-         when (not (llvmVersionSupported ver) && doWarn) $ putMsg logger $
-           "You are using an unsupported version of LLVM!" $$
-           "Currently only" <+> text (llvmVersionStr supportedLlvmVersionLowerBound) <+>
-           "up to" <+> text (llvmVersionStr supportedLlvmVersionUpperBound) <+> "(non inclusive) is supported." <+>
-           "System LLVM version: " <> text (llvmVersionStr ver) $$
-           "We will try though..."
          let isS390X = platformArch (llvmCgPlatform cfg)  == ArchS390X
          let major_ver = head . llvmVersionList $ ver
          when (isS390X && major_ver < 10 && doWarn) $ putMsg logger $
